content/blog/the-rise-of-special-project-infra.md

```diff
@@ -0,0 +1,190 @@
+---
+title: The Rise of Special Project _infra_
+date: 2022-08-12
+extra:
+  author: Ctem
+---
+
+_72 hours out. A bead of sweat slides from your brow, falls to the marred chassis of your local build server, and sizzles into mist, leaving a scant salt stain to tell the tale. It’s the start of the hottest Summer of Nix on record, and you have three days to [research](#phase-i-research), [provision](#phase-ii-provision), [configure](#phase-iii-configure), and [deploy](#phase-iv-deploy). With or without you, this lecture is going live._
+
+The deadline was 19 July, 2022; at five o’clock on that glistening Tuesday afternoon, one Eelco Dolstra – a living legend to those who understand – would take to the webcam from the verdant city of Utrecht to deliver a highly anticipated slice of fresh perspective on his now-19-year-old brainchild, [Nix](https://nixos.org/learn.html). It was the inaugural event in the premier Summer of Nix (SoN) Public Lecture Series. The hype was real, but so was our predicament: no self-hosted livestreaming infrastructure was yet in place.
```

Comment:
Is this a typo?
```suggestion
The deadline was 19 July, 2022; at five o’clock on that glistening Tuesday afternoon, one Eelco Dolstra – a living legend to those who understand – would talk to the webcam from the verdant city of Utrecht to deliver a highly anticipated slice of fresh perspective on his now-19-year-old brainchild, [Nix](https://nixos.org/learn.html). It was the inaugural event in the premier Summer of Nix (SoN) Public Lecture Series. The hype was real, but so was our predicament: no self-hosted livestreaming infrastructure was yet in place.
```

Reply:
“Take to the [platform/medium]" is an expression, where _platform/medium_ can be practically anything (e.g., stage, podium, mic, pen, quill). It’s a bit of literary flair to complement the energetic tone of the paragraph.

Reply:
Okay, please don't use idioms in technical writing. Most people who may be able to read your article ([1.5 B](https://en.m.wikipedia.org/wiki/List_of_languages_by_total_number_of_speakers#Top_languages_by_population)) don't know enough idiomatic English to make sense of them. (Sample size 1: 20 years of exposure and 10 years of daily English practice is apparently not enough.)

Possibly self-explanatory idioms are fine. See [controlling the narrative vs. framing the discourse](https://github.com/NixOS/nixos-summer/pull/38/files#diff-44d8314b85b5138271eaabb0c64e176694dbff2c03a4e29bd74e3fb90da1b162R36).